app/views/layouts/decidim/header/_main.html.erb

```diff
@@ -1,3 +1,14 @@
+<div class="main-bar brand-bar">
+  <div class="bcn-cat-link">
+    <a href="https://www.barcelona.cat/ca" class="bcnbrand-main-link" title="Barcelona" data-ga="barcelona.cat">www.barcelona.cat</a>
```

Comment:
I would remove the `data-ga` attribute if they didn't request us to include it. I would also include translations for the title.

You are not including the images properly. As we are using an ERB file, we can use the helpers provided by rails (`image_pack_tag` for the images or `link_to` for the links).

Reply:
@HHassig this is still pending. Please solve it before I can merge it